### PR TITLE
PDS and PDSE 8 character informational message

### PR DESCRIPTION
**Pull request by abdrabrcom**
#243 

Inroducing an informational message when using more than 8 characters for copybook names because of a limitation when searching PDS/PDSE dataset members

### CODE REVIEW

File: com.ca.lsp.cobol/lsp-core-cobol-parser/src/main/java/com/ca/lsp/core/cobol/preprocessor/sub/document/impl/CobolSemanticParserListenerImpl.java [hunk 1]
```diff
@@ -134,9 +136,21 @@ public void exitCopyStatement(@Nonnull CopyStatementContext ctx) {
...
 
+  private void checkCopybookNameLength(String copybookName, Position position) {
+    if (copybookName != null && copybookName.length() > 8) {
+      errors.add(
+          SyntaxError.syntaxError()
+              .severity(3)
+              .suggestion(String.format(COPYBOOK_OVER_8_CHARACTERS, copybookName))
+              .position(position)
+              .build());
+    }
```
Comment (zacanbrcom) [inline com.ca.lsp.cobol/lsp-core-cobol-parser/src/main/java/com/ca/lsp/core/cobol/preprocessor/sub/document/impl/CobolSemanticParserListenerImpl.java]: I saw that this routine is very similar as the method ```toSyntaxError```
I think is a good idea try to define a more general function and call it two times or even better delegate this responsibility to an appropriate class and made it more general and available for all the core and service module

Comment (temanbrcom) [reply com.ca.lsp.cobol/lsp-core-cobol-parser/src/main/java/com/ca/lsp/core/cobol/preprocessor/sub/document/impl/CobolSemanticParserListenerImpl.java]: This class is going to be deleted and its logic will be moved to `CobolSemanticParser`, so at least for me it is better to keep it like this and refactor later.

Comment (zacanbrcom) [reply com.ca.lsp.cobol/lsp-core-cobol-parser/src/main/java/com/ca/lsp/core/cobol/preprocessor/sub/document/impl/CobolSemanticParserListenerImpl.java]: ok

**Pull request merged**